int32 vx
int32 vy
int32 omega
---
uint8 result
